eWonConnector-gateway/src/main/java/org/imdc/ewon/comm/CommunicationManger.java

```diff
@@ -116,6 +116,23 @@ public EwonsData syncData(Long transactionId) throws Exception {
       return gson.fromJson(body, EwonsData.class);
    }
 
+   public void writeTag(String device, String tagName, String tagValue) throws Exception {
+      List<String> params = new ArrayList<>();
+      String directory = EwonConsts.T2M_DIR_GET + device + "/" + EwonConsts.T2M_DIR_RCGI;
+
+      params.add(EwonConsts.T2M_PARAM_TAGNAME1);
+      params.add(tagName);
+
+      params.add(EwonConsts.T2M_PARAM_TAGVALUE1);
+      params.add(tagValue);
+
+      params.add(EwonConsts.T2M_M2W_DEVKEY);
+      params.add(authInfo.getDevKey());
+
+      EwonUtil.httpGet(buildT2MCall(directory, EwonConsts.T2M_CALL_UPDATETAGFORM, params.toArray(new String[params.size()])));
```

Comment:
Probably just my personal preference, but for lines that have a lot of nested parentheses, it's easier to read and understand with some spaces. 

EwonUtil.httpGet( buildT2MCall( directory, EwonConsts.T2M_CALL_UPDATETAGFORM, params.toArray( new String[ params.size() ] ) ) );

Reply:
thats a tough one, i dont think google style has strict rules on that.  I use a bracket colorizer in my editor which color codes the brackets for each level.  I can change it if it helps people with other editors.

Reply:
I don't see strict rules either. I'm not sure other opinions, but that'ss just my preference. I did just get a bracket colorizer though, so its not a big deal.